src/routes/admin.js

```diff
@@ -0,0 +1,30 @@
+import express from 'express'
+import path from 'path'
+import groupBy from 'lodash/groupBy'
+import * as templateTask from '../database/queries/template_task'
+const router = express.Router()
+
+router.get('/template_task/get', function(req, res, next){
```

Comment:
once you rename the route to `api/admins/...` you should rename this to `/template_tasks`. You don't explicitly have to add the `method` (either get or post) in the route name.